assets/stylesheets/environment/tools/mixins/_grid-layout.scss

```diff
@@ -26,12 +26,18 @@
 ///   div {
 ///     @include grid-column(1 / 2, $float: right);
 ///   }
-@mixin grid-column($width, $full-width: tablet, $float: left) {
+@mixin grid-column($width, $full-width: desktop, $float: left) {
+  display: block;
   padding: 0 ($gutter / 2);
+  max-width: $measure;
```

Comment:
Been round the houses on this one.

Given that a set of grid columns *must* sit in a `.grid-row`, this approach actually makes sense to me - at least in this instance.

It feels more universal than adding lots of utility `div.measure` (or similar) everywhere. I'm also loath to start setting `max-width: $measure` on lots of components... 

This way it's really tied in to the behaviour of the grid. Which is actually what we want.

The other option is to always add `div.measure` or similar as a first child of every grid column that could:

a: snap to 100% width
b: conceivably be more than `$measure` wide when it's a column